src/ui/pages/equipment/location/Location.js

```diff
@@ -77,6 +81,36 @@ export default Location = (props) => {
         commentsComponent.current?.createCommentForNewEntity(location.code);
     }
 
+    const getTabGridRegions = () => {
+        const customTabGridRenderers = getCustomTabGridRenderers(applicationData);
+        return Object.entries(locationLayout.customGridTabs).map(([tabId, tab], index) => {
+            return ({
+                id: tab.tabDescription.replaceAll(' ','').toUpperCase(),
+                label: tab.tabDescription,
+                isVisibleWhenNewEntity: true,
+                maximizable: true,
+                render: ({ isMaximized }) =>
+                    <EAMGridTab
+                        screenCode={screenCode}
+                        tabName={tabId}
+                        objectCode={equipment.code}
+                        paramNames= {customTabGridParamNames}
+                        customRenderers={customTabGridRenderers}
+                        showGrid={isMaximized}
+                        rowCount={100}
+                        gridContainerStyle={isMaximized ? { height: `${document.getElementById('entityContent').offsetHeight - 220}px`} : {}}
+                    >   
+                    </EAMGridTab>
+                ,
+                column: 2,
+                order: 30 + 5 * index,
+                summaryIcon: GridOnIcon,
+                ignore: !tab.tabAvailable,             
+                initialVisibility: tab.alwaysDisplayed    
+            });
+        })
+    }
```

Comment:
This function is really similar throughout. Can we externalize it?

Reply:
@ricardo-j-almeida done